src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_CrustalInversionConfiguration.java

```diff
@@ -373,214 +193,19 @@ public static NZSHM22_CrustalInversionConfiguration forModel(InversionModels mod
 				.setSlipRateConstraintWt_normalized(slipRateConstraintWt_normalized)
 				.setSlipRateConstraintWt_unnormalized(slipRateConstraintWt_unnormalized)
 				.setSlipRateWeightingType(slipRateWeighting)
-				
-				.setMfdEqualityConstraints(mfdEqualityConstraints)
-				.setMfdInequalityConstraints(mfdInequalityConstraints)
+
+				.setMfdEqualityConstraints(mfdEqualityConstraints).setMfdInequalityConstraints(mfdInequalityConstraints)
 
 				// Rate Minimization config
 				.setMinimizationConstraintWt(minimizationConstraintWt)
 				.setMinimumRuptureRateFraction(minimumRuptureRateFraction)
-				.setMinimumRuptureRateBasis(minimumRuptureRateBasis)
-				.setInitialRupModel(initialRupModel);
+				.setMinimumRuptureRateBasis(minimumRuptureRateBasis).setInitialRupModel(initialRupModel);

```

Comment:
same here

Reply:
weird - I think it's the formatter at work ?